style.css

```diff
@@ -0,0 +1,459 @@
+}
+
+#light {
+  color:  #e51937;
+}
+
+#light::after {
+  content: '_______';
+  color: #e51937;
+  display: flex;
+  flex-diretion: column;
+  justify-content: center;
+}
+
+.header__line {
+  position: absolute;
+  left: 458px;
+  top: 50px;
+}
+
+.menu__icon {
+  width: 24px;
+  height: 15px;
+  margin: 0 5px;
+}
+
+.main__title {
+  font-size: 90px;
+  font-family: 'montserratsemibold';
+  margin: 40px 0 0 0;
+  padding: 0;
+  text-align: center;
+}
+
+.main__subtitle {
+  font: 15px 'montserratmedium';
+  margin: 0;
+  padding: 0;
+  text-align: center;
+}
+
+.main__pic {
+  width: 100%;
+  margin: -110px 0 0 0;
+	
+}
+
+.main__info {
+  display: flex;
+  justify-content: center;
+}
+
+.info__content {
+  display: flex;
+}
+
+.main__content {
+  margin: 30px 0;	
+}
+
+.main__content-item {
+  margin: 0 20px;
+  display: flex;	
+}
+
+.content {
+  display: flex;
+  align-items: center;
+  margin: 0 20px;
+}
+
+.content__item {
+  margin-right: 5px;
+  display: flex;	
+  flex-direction: column; 
+}
+
+
+.content__pic {
+  width: 46px;
+  height: 35px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.content__item-one {
+  font: 35px 'montserratsemibold';
+  margin: 0 5px 5px 0;
+  display: flex;
+  align-items: baseline;
+  text-align: left;
+}
+
+.content__one-big {
+  font: 35px 'montserratsemibold';
+  margin: 0 5px 0 0;
+}
+
+.content__one-small {
+  font: 20px 'montserratsemibold';	
+}
+
+.content__item-two {
+  font: 12px 'montserratregular';
+}
+
+button {
+  border: none;
+  display: flex;
+  align-items: center;
+  padding: 0 40px;
+  color: black;
+  border: 2px solid white;
+  font: 13px 'montserratbold';
+}
+
+.button__out {
+  border: none;
+  display: flex;
+  align-items: center;
+}
+
+button:hover {
+  opacity: 0.5;
+}
+
+.button__pic {
+  margin-right: 5px;
+  align-items: center;
+  display: flex;
+  justify-content: center;
+}
+
+.button__redwheel {
+  display: none;
+}
+
+.button__border {
+  display: flex;
+  align-items: center;
+  border: 2px solid #e51937;
+  color:#e51937;
+}
+.button__text {
+  text-transform: uppercase;
+}
+
+.main__pointer {
+  margin: 0 auto;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+
+/*
+section {
+display: flex;
+}
+
+.section__title {
+	color: #e51937;
+	font: 70px 'montserratsemibold';
+	position: absolute;
+}
+
+.article {
+width: 440px;
+	color: white;
+	background-color: #E51937;
+margin: 0;
+
+}
+
+.section__pic {
+float: left;
+}
+
+.article {
+	color: white;
+	background-color: #E51937;
+
+}
+
+.article__info {
+	margin: 20px 0 0 10px;
+	padding: 10px;
+
+}
+
+.article__charset {
+	margin: 0 0 10px;
+	display: flex; 	
+}
+
+.article__top {
+	margin: 20px 30px 40px 20px;
+}
+.article__title {
+	text-align: left;
+	padding: 10px;
+	font-size: 30px;
+}
+.article__text {
+	text-align: left;
+}
+
+.article__charset-item {
+	margin: 20px 10px;
+	
+}
+.charset {
+	display: flex;
+	align-items: center;
+	justify-content: center;	
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/tesla394/pullRequest?prid=5642156)